types/firestore.d.ts

```diff
@@ -1452,6 +1452,30 @@ declare namespace FirebaseFirestore {
    * API and the underlying Firestore v1 RPCs.
    */
   export const v1: {FirestoreClient: any, FirestoreAdminClient: any};
+
+  /**
+   * Status error code enums used by google-gax that correspond to the error
```

Comment:
Can you drop either "code" or "enums" (and probably google-gax, which is an internal library). My preference would be just "Status codes returned by Firestore's GRPC calls".